data/skmominali17.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "S K MOMIN ALI",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student, Learning Development",
+  "avatar": "https://github.com/skmominali17.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/SKMOMINALI20",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/skmominali17",
+      "icon": "FaGitHub"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "www.linkedin.com/in/s-k-momin-ali-737ba8220",
```

Not sure if `www.` links will work properly.
In my opinion, it's best to use `https://`.
```suggestion
      "url": "https://linkedin.com/in/s-k-momin-ali-737ba8220",
```

---

Thanks